src/hotspot/share/gc/z/zArguments.cpp

```diff
@@ -219,7 +220,20 @@ size_t ZArguments::heap_virtual_to_physical_ratio() {
 }
 
 CollectedHeap* ZArguments::create_heap() {
-  return new ZCollectedHeap();
+  // ZCollectedHeap has an alignment >= ZCacheLineSize, which may be larger than
```

Comment:
The `ZCollectedHeap has an alignment >= ZCacheLineSize,` had me reading the sentence a couple of times. I think it would be easier to read if this used words instead of >=.